Tests/FluentAssertions.Specs/Types/MethodBaseAssertionSpecs.cs

```diff
@@ -57,6 +56,37 @@ public void When_asserting_a_void_method_returns_string_it_fails_with_a_useful_m
                              "error message, but it is \"System.Void\".");
         }
 
+        [Fact]
+        public void When_subject_is_null_return_should_fail()
```

Comment:
We really need to improve this naming convention or the grouping of tests, because using the literal name in the test name is what I want to avoid. 

It _would_ be fine if it said: When subject is null, `return` should fail. But without the markup, it's difficult to read as a sentence.